library/src/main/java/com/google/android/exoplayer/text/subrip/SubripParser.java

```diff
@@ -58,6 +58,10 @@ public SubripSubtitle parse(InputStream inputStream) throws IOException {
     String currentLine;
 
     while ((currentLine = reader.readLine()) != null) {
+      // Skip blank lines.
+      if (currentLine.length() == 0)
```

Comment:
This could have used `TextUtils.isEmpty` but since we already know it's not null, I thought simplest to just check it's length directly.